examples/speech_to_text/visualize.py

```diff
@@ -0,0 +1,80 @@
+import os
+import pandas as pd
+import argparse
+import pprint
+
+
+def read_scores_from_folder(folder_path):
+    score_file_path = os.path.join(folder_path, "scores.tsv")
+    if os.path.isfile(score_file_path):
+        with open(score_file_path, "r") as f:
+            contents = [line.strip() for line in f.read().split("\n") if line.strip()]
+        return contents
+    else:
+        return None
+
+
+def read_scores_files(output_folder):
+    all_contents = []
+
+    if not os.path.isdir(output_folder):
+        raise ValueError("Output folder does not exist")
```

Comment:
Can't we simply make the folder instead of throwing an error?

Reply:
We cannot make the folder because that's where we want to read all the results from. So if the folder does not exist, there is nothing to read